packages/graphql-mesh-server/lib/metrics.ts

```diff
@@ -355,7 +356,8 @@ export class PerformanceMetrics extends Construct {
 
     // Create the dashboard
     new Dashboard(this, "dashboard", {
-      dashboardName: "Mesh-Dashboard",
+      dashboardName:
+        props.dashboardName ?? Stack.of(this).stackName + "-Mesh-Dashboard",
```

Comment:
I think defaulting to undefined is fine. CDK will just use an auto generated dashboard name

Reply:
Updated now :+1: